packages/synapse-react-client/src/assets/mui_components/Docker.tsx

```diff
@@ -3,7 +3,7 @@ import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon'
 
 const Docker = (props: SvgIconProps) => {
   return (
-    <SvgIcon {...props}>
+    <SvgIcon viewBox="-2 0 26 17" {...props}>
```

Comment:
SVGs are defined by the paths and other elements that are nested in the `svg` tag, and the `viewBox` attribute defines the area that can be seen. These changes just center the icons. This was done by hand/trial-and-error, but we could have also used a tool like Inkscape, or just asked the design team for centered versions.